.github/workflows/sync-figma-tokens.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync tokens to Figma
+
+on:
+  push:
+    branches:
+      - production
+    paths:
+      - "tokens/movistar.json"
```

Comment:
I think you can do something like "tokens/*.json" or similar. This will prevent you from updating this in the future

Reply:
If we add new skin and we dont want to sync with Figma, just don't include in `.mjs` script

Reply:
At the end, we will keep "paths" for each brand.json file to avoid run the script with unofficial .json files